src/Microsoft.PowerShell.Commands.Utility/commands/utility/Tee-Object.cs

```diff
@@ -85,6 +86,15 @@ public string Variable
 
         private string _variable;
 
+        /// <summary>
+        /// Sets the encoding to use when writing the file.
+        /// </summary>
+        [Parameter]
+        [ArgumentToEncodingTransformationAttribute]
+        [ArgumentEncodingCompletionsAttribute]
+        [ValidateNotNullOrEmpty]
+        public Encoding Encoding { get; set; } = ClrFacade.GetDefaultEncoding();
```

Comment:
I guess CIs fail because we need explicitly make the parameter optional (Mandatory = false).